main.js

```diff
@@ -113,10 +116,19 @@ async function init(platform) {
 }
 
 async function main() {
+    const parser = new ArgumentParser({description: 'main parser'});
+    parser.addArgument(['-l', '--locale'], {
+        required: false,
+        defaultValue: 'en-US',
+        help: `BGP 47 locale tag of the language to evaluate (defaults to 'en-US', English)`
+    });
+    const args = parser.parseArgs();
```

Comment:
Command line arguments for the server are not useful, because the server is not run on the command line normally, it's run in a docker or systemd unit. Use environment variables instead.